Gems/Multiplayer/Code/Source/Debug/MultiplayerConnectionViewportMessageSystemComponent.h

```diff
@@ -12,19 +12,27 @@
 #include <AzCore/Component/Component.h>
 #include <AzFramework/Font/FontInterface.h>
 #include <Multiplayer/IMultiplayer.h>
-#include <Multiplayer/IMultiplayerConnectionViewportMessage.h>
+#include <Multiplayer/MultiplayerEditorServerBus.h>
 
 
 namespace Multiplayer
 {
     //! System component that draws viewport messaging as the editor attempts connection to the editor-server while starting up game-mode.
     class MultiplayerConnectionViewportMessageSystemComponent final
         : public AZ::Component
-        , public AZ::Interface<IMultiplayerConnectionViewportMessage>::Registrar
         , public AZ::RPI::ViewportContextNotificationBus::Handler
+        , MultiplayerEditorServerNotificationBus::Handler
     {
     public:
-        AZ_COMPONENT(MultiplayerConnectionViewportMessageSystemComponent, "{7600cfcf-e380-4876-aa90-8120e57205e9}", IMultiplayerConnectionViewportMessage);
+        static constexpr char CENTER_VIEWPORT_DEBUG_TITLE[] = "Multiplayer Editor";
```

Comment:
The all upper-case thing is code convention for macros only I thought? #define SOMETHING_THATS_A_MACRO some_value

Reply:
I also wonder if we have any plans for a localizable string database for tooling like the editor..

Reply:
Good point, I believe these should all be upper camel case.

Reply:
Good call out! That's what I thought too, but a search for "constexpr char" inside of Gems/Multiplayer were all caps T.T
The other constexpr's are normal casing though, I bet that was just a mistake... I'll keep proper standards and stick with normal casing. Fixed!